### PR TITLE
zh-translation:content/zh/docs/ops/diagnostic-tools/multicluster/index.md

### PR DESCRIPTION
**Pull request by mrshengzyzy**
任务列表缺失次任务

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by kebe7jun** [comment]
hello, 可以先修复一下 lint 么？

**Comment by mrshengzyzy** [comment]
@kebe7jun 我不知道如何修复此问题。
lint 指出使用了强调代替标题，实际上此处就应该是强调而不是标题，属于误报？

**Comment by Xunzhuo** [comment]
@mrshengzyzy to solve this question, please use markdown headers instead of emphasized text to denote sections

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+NAME                      TYPE           CLUSTER-IP    EXTERNAL-IP      PORT(S)                                                           AGE
+istio-eastwestgateway    LoadBalancer   10.8.17.119   <PENDING>        15021:31781/TCP,15443:30498/TCP,15012:30879/TCP,15017:30336/TCP   76m
+{{< /text >}}
+
+如果 `EXTERNAL-IP` 卡在 `<PENDING>` 状态, 则说明环境可能不支持 `LoadBalancer` 服务。在这种情况下，可能需要自定义服务的 `spec.exteralIPs` 部分，手动为网关提供集群外可达的 IP。
+
+如果外部 IP 存在，请检查 `topology.istio.io/network` 标签的值是否正确。如果不正确，请重新安装网关，并确保在生成脚本上设置 --network 标志。
+
+**无法确定客户端 Pod 或服务器 Pod 的网络：**
+
```
Comment (Xunzhuo) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:140]: ###  无法确定客户端 Pod 或服务器 Pod 的网络：

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+10.0.6.13:5000                   HEALTHY     OK                outbound|5000||helloworld.sample.svc.cluster.local
+{{< /text >}}
+
+多网络中模型中，我们期望其中一个端点 IP 与远程群集的东西向网关(east-west gateway)公网 IP 匹配。看到多个 Pod IP 说明存在以下两种情况:
+
+* 无法确定远程网络的网关地址。
+* 无法确定客户端 Pod 或服务器 Pod 的网络。
+
+**无法确定远程网络的网关地址：**
+
```
Comment (Xunzhuo) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:126]: ###  无法确定远程网络的网关地址：

File: content/zh/docs/ops/diagnostic-tools/multicluster/index.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+[本地负载均衡](/zh/docs/tasks/traffic-management/locality-load-balancing/failover/#configure
+-locality-failover) 总是引导客户端访问最近的服务。如果集群分布于不同地理位置(地区/区域)，本地负载均衡将优先选用本地实例提供服务，这与预期相符。而如果禁用了本地负载均衡或者是集群处于同一地理位置，那就可能还存在其他问题。
+
+### 受信配置{#trust-configuration}
+
+与集群内通信一样，跨集群通信依赖于代理之间公共的且可信任的根证书颁发机构(root)。默认情况下 Istio 使用自身单独生成的根证书颁发机构。对于多集群的情况，我们必须手动配置公共的且可信任的根证书颁发机构。阅读下面的 **Plug-in Certs** 章节或者参考[身份和信任模型](/zh/docs/ops/deployment/deployment-models/#identity-and-trust-models)了解更多细节。
+
+**证书插件：**
+
```
Comment (Xunzhuo) [inline content/zh/docs/ops/diagnostic-tools/multicluster/index.md:39]: ### 证书插件：

**Review by Xunzhuo — COMMENTED** [review]
Please use markdown headers instead of emphasized text to denote sections

**Review by Xunzhuo — COMMENTED** [review]
To solve this question, please use markdown headers instead of emphasized text to denote sections

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**